code/game/machinery/doors/airlock_electronics.dm

```diff
@@ -3,100 +3,102 @@
 	icon = 'icons/obj/doors/door_assembly.dmi'
 	icon_state = "door_electronics"
 	w_class = WEIGHT_CLASS_SMALL
-	materials = list(MAT_METAL=50, MAT_GLASS=50)
+	materials = list(MAT_METAL = 50, MAT_GLASS = 50)
 	origin_tech = "engineering=2;programming=1"
 	req_access = list(ACCESS_ENGINE)
 	toolspeed = 1
 	usesound = 'sound/items/deconstruct.ogg'
-	var/list/conf_access = null
-	var/one_access = 0 //if set to 1, door would receive req_one_access instead of req_access
-	var/const/max_brain_damage = 60 // Maximum brain damage a mob can have until it can't use the electronics
-	var/unres_sides = 0
+	/// List of accesses currently set
+	var/list/selected_accesses = list()
+	/// Is the door access using req_one_access (TRUE) or req_access (FALSE)
+	var/one_access = FALSE
+	/// An associative list containing all station accesses. Includes their name and access number.
+	var/static/list/door_accesses_list = list()
+	/// Maximum brain damage a mob can have until it can't use the electronics
+	var/const/max_brain_damage = 60
+	/// Which direction has unrestricted access (e.g. medbay doors from the inside)
 	var/unres_direction = null
```

Comment:
Does not explain which direction you can come FROM or which direction you can go TOWARDS.
Definitely needs a better comment, but would be great if the name of the var had an indication of this (although the latter might be out of scope).

Reply:
How's `unres_access_from` for a var name?
I've already touched airlock code so it's within scope.

Reply:
`_from` is good. It's not super eloquent of course, but it's sufficiently unconfusing.